torch/onnx/utils.py

```diff
@@ -298,7 +320,7 @@ def _trace_and_get_graph_from_model(model, args, training):
     return trace_graph, torch_out
 
 
-def _model_to_graph(model, args, verbose=False, training=False,
+def _model_to_graph(model, args, verbose=False, training=None,
```

Comment:
the same applies here as well. `training` parameter is not really used inside the method. It all depends on the caller of `_model_to_graph` to properly set training mode, by calling `with select_model_mode_for_export(model, training):`, otherwise results may be unexpected.